### PR TITLE
fix/gui upload

### PR DESCRIPTION
**Pull request by JarbasAl**
raises the maz sixe of bus messages to 100MB

this should not be an issue, but i am cautious of allowing a large value here, ideally the upload should be splitting the message into several smaller ones instead

this PR is mostly meant for discussion and to get any existing users running for early testing, 100MB does not seem a sane value to me!

### CODE REVIEW

File: ovos_messagebus/event_handler.py [hunk 1]
```diff
@@ -33,13 +33,23 @@ def __init__(self, application, request, **kwargs):
...
     def on_message(self, message):
-        LOG.debug(message)
+
         try:
             deserialized_message = Message.deserialize(message)
         except Exception:
             return
 
+        filter_ogs = ["gui.status.request"]
+        if deserialized_message.msg_type not in filter_ogs:
```
Comment (JarbasAl) [inline ovos_messagebus/event_handler.py:48]: log spam https://paste.api-zulu.com/?f0bcef52305ca135#5hcVD9fd8vJBbcyNZWJyGi2gkb3XcCfEpNg3ouyE1BNr

**Comment by goldyfruit** [comment]
> 
> this should not be an issue, but i am cautious of allowing a large value here, ideally the upload should be splitting the message into several smaller ones instead
> 

I agree, the upload should be split as it will reduce the CPU consumption as well _(when trying to upload the CPU when crazy)_.

**Comment by goldyfruit** [comment]
Maybe default could stay 10MB and when required, it could be overrided via a setting from `mycroft.conf`.

**Comment by JarbasAl** [comment]
> Maybe default could stay 10MB and when required, it could be overrided via a setting from `mycroft.conf`.

agree with this, change made

the 10MB limit stays as a default, but mycroft.conf can be changed now

**Pull request merged**